dependencies.props

```diff
@@ -89,7 +89,7 @@
     <NuGetDependency Include="NuGet.RuntimeModel"/>
     <NuGetDependency Include="NuGet.Versioning"/>
     <StaticDependency Include="@(NuGetDependency)">
-      <Version>4.3.0</Version>
+      <Version>4.4.0</Version>
```

Comment:
We need to avoid bumping nuget to 4.4.0 without extensive testing

Reply:
What's the current version in  2.0 cli? 4.3.0 right?

Reply:
Yes, which is why I believe this will cause failures.